quickhash.c

```diff
@@ -331,6 +339,9 @@ PHP_MSHUTDOWN_FUNCTION(quickhash)
 
 PHP_RINIT_FUNCTION(quickhash)
 {
+	#if defined(COMPILE_DL_QUICKHASH) && defined(ZTS) && PHP_VERSION_ID >= 70000
+		ZEND_TSRMLS_CACHE_UPDATE();
+	#endif
```

Comment:
The ```#if``` and ```#endif``` must be at the first column, as some compilers don't like it this way.